src/main.rs

```diff
@@ -77,9 +77,7 @@ fn main() -> Result<()> {
 
     event_loop.run(move |event: Event<'_, ()>, _, control_flow| {
         state.platform.handle_event(&event);
-        if captures_event(&state.platform, &event) {
-            info!("captured");
```

Comment:
If you could remove that 
`use log::info;` on line 40, clippy will be happy and this can be merged

Reply:
Just did it :)